console_run.py

```diff
@@ -0,0 +1,159 @@
+import argparse
+import os
+import signal
+import warnings
+from datetime import datetime, timedelta
+
+from LatiaoAid import LatiaoAid
+from Logger import Logger
+
+
+def parse_args() -> argparse.Namespace:
+    """Parse the command line arguments for the `LatiaoAid` binary.
+
+    :return: Namespace with parsed arguments.
+    """
+    parser = argparse.ArgumentParser(
+        description="Collects 辣条/亲密度 on live.bilibili.com using selenium webdriver"
+    )
+
+    parser.add_argument(
+        "-r", "--room", type=int, help="default room", default=22198526,
+    )
+
+    # runtime behaviors
+    parser.add_argument(
+        "--headless", help="do not show the browser", action="store_true",
+    )
+
+    parser.add_argument(
+        "-d",
+        "--disable-image",
+        help="do not show the video content",
+        action="store_true",
+    )
+
+    # log settings
+    parser.add_argument(
+        "-v", "--verbose", help="do not print log to console", action="store_true",
+    )
+
+    parser.add_argument(
+        "-l", "--log", help="save log to the log file", action="store_true",
+    )
+
+    # arg check
+    parser.add_argument(
```

Comment:
Why the validation of arguments is optional?

Reply:
Although check_args is not expensive now, future is full of possibilities. This parameter allows confident users to skip the check, and let errors thrown in subsequent operations.